### PR TITLE
fix a problematic url

### PR DESCRIPTION
**Pull request by szcf-weiya**
fix a problematic url
image

### CODE REVIEW

**Comment by szcf-weiya** [comment]
Another related problem is that the hover state on the URL link `a:hover`  which is rendered from `< >` will also enlarge the appending text, as shown in the following gif,
Peek 2021-07-17 11-29

but not sure how to properly fix it. 

Here are some of my guesses, note that one URL link that rendered from `[]()` seems OK, such as 
Peek 2021-07-17 11-30
so it might work to replace `<SOME-URL>` with `[SOME-URL](SOME-URL)`

Or update the CSS (if possible), which seems more convenient and fast, I mean, no need to correct each such link one by one.

File: content/post/2016-07-02-statistics-on-article-titles.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ forum_id: 419150
 此外，我们希望选出来的词项具有一定的代表性，所以在构造词频矩阵的时候，我们要求每一个词项在文章集合中至少出现过两次。
 这样就避免了一些特殊的词语搭配因为出现在某一篇访问量极高的文章中而被挑选出来。
 
-在R中，可以使用`text2vec`包(https://cran.r-project.org/web/packages/text2vec/index.html)很方便地完成这一过程，代码如下：
+在R中，可以使用`text2vec`包 <https://cran.r-project.org/web/packages/text2vec/index.html> 很方便地完成这一过程，代码如下：
```
Comment (yihui) [inline content/post/2016-07-02-statistics-on-article-titles.md:89]: 我建议把链接直接加到文字上：

```suggestion
在R中，可以使用 [`text2vec`包](https://cran.r-project.org/web/packages/text2vec/index.html)很方便地完成这一过程，代码如下：
```

**Review by yihui — APPROVED** [review]
你提到的第二个问题我觉得应该是视觉上的假象：无论是裸链接，还是加在文本上的链接，其 CSS 样式都是一样的设定，只不过后者加粗对后面的文本流的影响没有前者那么明显，因为后者比较短，前者比较长（会把后面的文本更多地挤下去）。

我有点倾向于把 `a:hover` 的加粗样式取消掉。不知其他人怎么看。

谢谢！

**Comment by szcf-weiya** [comment]
这么一说后面文字加粗确实只是假象

**Pull request merged**